scripts/ci.go

```diff
@@ -0,0 +1,71 @@
+// +build none
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	GOLANGCI_VERSION = "github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.1"
+)
+
+func main() {
+
+	log.SetFlags(log.Lshortfile)
+
+	if _, err := os.Stat(filepath.Join("scripts", "ci.go")); os.IsNotExist(err) {
```

Comment:
missing else check

Reply:
not really. There is no Else check. What do you want to check ? maybe I am not seeing it.